services/web/server/src/simcore_service_webserver/rest_handlers.py

```diff
@@ -9,12 +9,32 @@
 
 from ._meta import __version__, api_version_prefix
 from .application_settings import APP_SETTINGS_KEY
+from .rest_healthcheck import HealthCheckFailed, HeathCheck
 
 log = logging.getLogger(__name__)
 
 routes = web.RouteTableDef()
 
 
+@routes.get(f"/{api_version_prefix}/health", name="check_health")
+async def check_health(request: web.Request):
+
+    healthcheck: HeathCheck = request.app[HeathCheck.__name__]
+
+    if timeout := request.app[APP_SETTINGS_KEY].SC_HEALTHCHECK_TIMEOUT:
+        # Let's cancel check if goes 10% over healtcheck timeout
+        timeout *= 1.1
```

Comment:
Why not already add 10% when defining the`SC_HEALTHCHECK_TIMEOUT`?

Reply:
that is what i do above, right?
In any case, after discussion with you, i set to exactly that value